pysat/_constellation.py

```diff
@@ -136,6 +279,62 @@ def bounds(self, value=None):
 
         return
 
+    @property
+    def date(self):
+        """Date for loaded data."""
+        if len(self.index) > 0:
+            return utils.time.filter_datetime_input(self.index[0])
+        else:
+            return None
+
+    @property
+    def empty(self):
+        """Boolean flag reflecting lack of data, True if there is no Instrument
```

Comment:
I actually posted a couple incorrect suggested changes since I didn't properly get this at first. I tried the code before reading the docstring 🐒

Perhaps `empty` isn't the best word here since you are really checking if it is full (all_inst=True).

I don't think there will ever be good data for both ivm 'a' and 'b' in the ICON constellation so that one would always be empty.

Reply:
See the above discussion for the `empty` flag.  Obviously, the name `empty` was used for compatibility with Instrument.

Reply:
I'm with you on having a compatible `empty` but my built in expectation was/is different than how it currently operates.